internal/statemachine/classic_states.go

```diff
@@ -1255,6 +1259,29 @@ func (stateMachine *StateMachine) populateClassicRootfsContents() error {
 	return nil
 }
 
+// Set a default locale if one is not configured beforehand by other customizations
+func (stateMachine *StateMachine) setDefaultLocale() error {
+	classicStateMachine := stateMachine.parent.(*ClassicStateMachine)
+
+	defaultPath := filepath.Join(classicStateMachine.tempDirs.chroot, "etc", "default")
+	localePath := filepath.Join(defaultPath, "locale")
+	localeBytes, err := osReadFile(localePath)
+	if err == nil && strings.Contains(string(localeBytes), "LANG=") {
```

Comment:
It's a pain but this should really check for the regex "^LANG=" right? This would match "#LANG=" ...